include/examples/simple_robot.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file simple_robot.h
+ * @author J. Diego Caporale
+ * @brief A simple RobotInterface derived class example. Here is where you would implement any state updates,
+ *        behaviors, or control system
+ * @date 2022-07-19
+ *
+ * @copyright Copyright (c) 2022
+ *
+ */
+
+#include "kodlab_mjbots_sdk/robot_interface.h"
+
+class SimpleRobot : virtual public kodlab::RobotInterface
+{
+
+    using kodlab::RobotInterface::RobotInterface;
+
+public:
+    int mode = 0; // Member variables encode whatever added state we need
+    // Set up robot update function for state and torques
+    void Update() override
+    {
+        std::vector<float> torques(num_joints_, 0);
```

Comment:
I don't think this is what we want in the update as an example for people to pattern match. Maybe give it a leg and use that??

Reply:
Will add better example code

Reply:
#36